whatToDad_site/whatToDad_app/templates/index.html

```diff
@@ -1,101 +1,100 @@
-    }
+  .rating>label {
+    position: relative;
+    width: 1em;
+    font-size: 3vw;
+    color: #FFD600;
+    cursor: pointer;
+  }
 
-    .rating > label:hover:before,
-    .rating > label:hover ~ label:before {
-      opacity: 1 !important;
-    }
+  .rating>label::before {
+    content: "\2605";
+    font-size: 3vw;
+    position: absolute;
+    opacity: 0;
+  }
 
+  .rating>label:hover:before,
+  .rating>label:hover~label:before {
+    opacity: 1 !important;
+  }
 </style>
 
-    <header class="masthead" >
-            <div class="overlay"></div>
-            <div class="container">
-              <div class="row">
-                <div class=" col-md-8 col-md-10 mx-auto">
-                  <div class="site-heading">
-                    <h3 class=" site-heading my-4 mt-3 text-white">This weeks top Fatherly Bond activity</h3>
-                    <img src="{% static 'fatherSon.jpg' %}">
-                  </p>
-                </div>
-                    </div>
-                  </div>
-                </div>
-            </div>
-              </header>
-        
-            <div class="container">
-                <div class="row">
-      
-              <!-- Blog Entries Column -->
-              <div class="col-md-8 mt-3 left">
-                    {% for post in post_list %}
-                <div class="card mb-4" >
-                  <div class="card-body">
-                    <h2 class="card-title">{{ post.title }}</h2>
-                    <div class="rating">
-                      <h3>Like Post </h3>
-                      <input type="radio" name="rating" value="1" id="1"><label for="1">☆</label>
-                    </div>
-                    <p class="card-text">{{post.content|slice:":200" }}</p>
-                    <a href="{% url 'post_detail' post.slug  %}" class="btn btn-primary">Read More &rarr;</a>
-                  </div>
-                 
-                </div>
-                {% endfor %}
-            </div>
-                {% block sidebar %}
-                {% include 'sidebar.html' %}
-                {% endblock sidebar %}
-            </div>
+<header class="masthead">
+  <div class="overlay"></div>
+  <div class="container">
+    <div class="row">
+      <div class=" col-md-8 col-md-10 mx-auto">
+        <div class="site-heading">
+          <h3 class=" site-heading my-4 mt-3 text-white">This weeks top Fatherly Bond activity</h3>
+          <img src="{% static 'fatherSon.jpg' %}">
         </div>
-{%endblock%}
+      </div>
+    </div>
+  </div>
+  </div>
+</header>
+
+<div class="container">
+  <div class="row">
+
+    <!-- Blog Entries Column -->
+    <div class="col-md-8 mt-3 left">
+      {% for post in post_list %}
+      <div class="card mb-4">
+        <div class="card-body">
+          <h2 class="card-title">{{ post.title }}</h2>
+          <div class="rating">
+            <h3>Like Post </h3>
+            <input type="radio" name="rating" value="1" id="1"><label for="1">☆</label>
```

Comment:
Also, because you are generating these cards in a loop, the `id`s and `for` are duplicated and therefore aren't attached to each other 1-1. If you need to generate unique names, you should use something unique to the post, similar to how you need `key`s when generating a list in React.

Reply:
I believe I have corrected this by adding the author id as a unique value. Please advise.